README.md

```diff
@@ -76,10 +76,10 @@ Mach works with the new self-hosted Zig compiler only. This means you'll need to
 
 Currently we test using `0.10.0-dev.3952+9e070b653`. Newer is generally better, but you can download this version here:
```

Comment:
Hm, I should remove this line. Looks like the script/regex doesn't update it correctly.